tt_metal/soc_descriptors/blackhole_simulation_1x2_arch.yaml

```diff
@@ -6,7 +6,7 @@ arc:
   []
 
 pcie:
-  []
+  [ 0-0 ]
```

Comment:
I think we should leave pcie empty in the soc desc yamls because there are no pcie cores in the sim setups. We could define `PCIE_NOC*` to avoid compilation issues but ignore it on device

Reply:
Makes sense, removed.